en/development/configuration.rst

```diff
@@ -626,7 +626,24 @@ This means for example, that if you need to change your database, you'll just
 need to modify a DATABASE_URL variable on your host configuration without the
 need to change it in your source code.
 
-As you can see in your **app.php**, the following variables are concerned:
+Because CakePHP will automatically prefer a setting exposed as an environment
+variable over its peer setting in ``app.php`` enabling environment variables is
```

Comment:
You are absolutely correct, I just realized I made a (huge) thinking error here caused by the fact that I had the @josegonzalez dotenv package already installed.

I will update (and write down that I shouldn't attempt rush jobs anymore)